[help wanted] Sirupsen caused installation to fail

### Ⅰ. Issue Description

```shell
➜  pouch git:(master) ✗ make install
GOOS=linux go build -o pouchd
daemon/mgr/network.go:17:2: case-insensitive import collision: "github.com/alibaba/pouch/vendor/github.com/Sirupsen/logrus" and "github.com/alibaba/pouch/vendor/github.com/sirupsen/logrus"
make: *** [Makefile:23: server] Error 1
```

logrus organization's name was changed to lower-case.If we use the upper-case import :`Sirupsen/logrus`,conflicts would happen due to case sensitivity.

### Ⅱ. Describe what happened


### Ⅲ. Describe what you expected to happen


### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1.
2.
3.

### Ⅴ. Anything else we need to know?

https://github.com/sirupsen/logrus#case-sensitivity
https://github.com/sirupsen/logrus/pull/384

### Ⅵ. Environment:

- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/daemon/mgr/network.go
+++ b/daemon/mgr/network.go
@@ -14,7 +14,6 @@ import (
 	"github.com/alibaba/pouch/pkg/meta"
 	"github.com/alibaba/pouch/pkg/randomid"
 
-	netlog "github.com/Sirupsen/logrus"
 	"github.com/docker/go-connections/nat"
 	"github.com/docker/libnetwork"
 	nwconfig "github.com/docker/libnetwork/config"
@@ -411,15 +410,15 @@ func (nm *NetworkManager) getNetworkSandbox(id string) libnetwork.Sandbox {
 
 func initNetworkLog(cfg *config.Config) {
 	if cfg.Debug {
-		netlog.SetLevel(netlog.DebugLevel)
+		logrus.SetLevel(logrus.DebugLevel)
 	}
 
-	formatter := &netlog.TextFormatter{
+	formatter := &logrus.TextFormatter{
 		ForceColors:     true,
 		FullTimestamp:   true,
 		TimestampFormat: "2006-01-02 15:04:05.000000000",
 	}
-	netlog.SetFormatter(formatter)
+	logrus.SetFormatter(formatter)
 }
 
 func endpointOptions(n libnetwork.Network, nwconfig *apitypes.NetworkSettings, epConfig *apitypes.EndpointSettings) ([]libnetwork.EndpointOption, error) {

--- a/network/mode/bridge/bridge.go
+++ b/network/mode/bridge/bridge.go
@@ -9,13 +9,13 @@ import (
 
 	"github.com/alibaba/pouch/apis/types"
 	"github.com/alibaba/pouch/daemon/mgr"
-	"github.com/alibaba/pouch/extra/libnetwork/Godeps/_workspace/src/github.com/Sirupsen/logrus"
 	"github.com/alibaba/pouch/network"
 	"github.com/alibaba/pouch/pkg/errtypes"
 
 	"github.com/docker/libnetwork/drivers/bridge"
 	"github.com/docker/libnetwork/netlabel"
 	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 	"github.com/vishvananda/netlink"
 )
 

--- a/registry/auth.go
+++ b/registry/auth.go
@@ -10,7 +10,7 @@ import (
 
 	"github.com/alibaba/pouch/apis/types"
 
-	"github.com/Sirupsen/logrus"
+	"github.com/sirupsen/logrus"
 )
 
 // challengeClient defines a client to manage challenge–response authentication.

--- a/volume/core.go
+++ b/volume/core.go
@@ -4,7 +4,6 @@ import (
 	"fmt"
 	"reflect"
 
-	"github.com/alibaba/pouch/extra/libnetwork/Godeps/_workspace/src/github.com/Sirupsen/logrus"
 	"github.com/alibaba/pouch/pkg/client"
 	metastore "github.com/alibaba/pouch/pkg/meta"
 	"github.com/alibaba/pouch/volume/driver"
@@ -13,7 +12,7 @@ import (
 	"github.com/alibaba/pouch/volume/types/meta"
 
 	"github.com/pkg/errors"
-	log "github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus"
 )
 
 // Core represents volume core struct.
@@ -45,7 +44,7 @@ func NewCore(cfg Config) (*Core, error) {
 		},
 	})
 	if err != nil {
-		log.Errorf("failed to create volume meta store: %v", err)
+		logrus.Errorf("failed to create volume meta store: %v", err)
 		return nil, err
 	}
 
@@ -206,7 +205,7 @@ func (c *Core) ListVolumeName(labels map[string]string) ([]string, error) {
 			return nil, errors.Wrap(err, "List volume's name")
 		}
 
-		log.Debugf("List volume URL: %s, labels: %s", url, labels)
+		logrus.Debugf("List volume URL: %s, labels: %s", url, labels)
 
 		if err := client.New().ListKeys(url, &names); err != nil {
 			return nil, errors.Wrap(err, "List volume's name")

